tests/test_visitors/test_ast/test_blocks/test_control_variables.py

```diff
@@ -43,6 +43,14 @@ def wrapper():
     print(wrapper)
 """
 
+correct_for_multi_loops = """
```

Comment:
Please, let's add some tests: I guess we need to test that `for` can reuse the variable this way. But other types of blocks cannot. Let's say that two `for` loops are ok, but a `for` loop followed by `with` block with the same variable names is not ok.

```python
wrong_loop_variable_reuse1 = """
def wrapper():
    for my_var in range(3):
        ...
    with some() as my_var:
        ...
"""
```

Reply:
I have added the test you included, and added a variable that checks whether the block variable is reused in other types of blocks.